Feat:#5 cicd

syjdjr opened the pull request:
## #️⃣연관된 이슈
> #5 

## 📝작업 내용
> 이번 PR에서 작업한 내용을 간략히 설명해주세요(이미지 첨부 가능)
CI/CD 진행했어요

### 스크린샷 (선택)
![image](https://github.com/user-attachments/assets/3bad7e61-bafa-44b9-803e-5165e42f67d4)

## 💬리뷰 요구사항(선택)
> 리뷰어가 특별히 봐주었으면 하는 부분이 있다면 작성해주세요
>

syjdjr committed:
feat: #5 CI/CD 코드 작성

- githubAction 을 이용한 CI/CD

syjdjr committed:
feat: #5 dockerfile 작성

syjdjr committed:
feat: #5 CI/CD 작성

syjdjr committed:
feat: #5 CI/CD

syjdjr committed:
feat: #5 CI/CD health check 구현

syjdjr committed:
feat: #5 CI/CD health check 구현

syjdjr committed:
feat: #5 CICD 수정 중

syjdjr committed:
feat: #5 CICD 수정 중

syjdjr committed:
feat: #5 CICD 수정 중

syjdjr committed:
feat: #5 CICD 수정 중

syjdjr committed:
feat: #5 CICD 수정 중

syjdjr committed:
feat: #5 CI/CD 구축

- develop 브랜치

yeonjy commented on build/libs/be-0.0.1-SNAPSHOT.jar:
빌드된 파일은 깃에 올리지 않는게 좋을 것 같습니당..!
로컬이랑 배포 환경이 달라서 호환성 문제가 발생할 수도 있을 것 같아요! 그리고 커밋할 때 충돌이 발생할 수도 있을 것 같습니다!
CI/CD 과정에서 빌드하는 것은 어려우실까용..?

syjdjr replied:
CI/CD 과정에서 빌드 하도록 코드 작성 진행된 상태라 가능합니다!
리뷰 주신 빌드된 .jar파일은 확인 하려고 test 용으로 진행했던 부분이라 .gitignore 처리해서 진행하겠습니당
회의 때 추가 진행된 부분이랑 리뷰 내용 반영해서 코드 정리 후에 내일 오후 중으로 반영하겠습니다!☺️☺️

yeonjy reviewed (commented):
고생하셨습니당 :)

syjdjr committed:
feat: #5 CI/CD 구축 추가 수정

syjdjr committed:
feat: #5 Cors 설정

syjdjr committed:
feat: #5 CICD 수정 사항 반영

- yml 파일 추가
- 회의 내용 추가 (-> security 진행 시 cors 활성화 필요)
- test 브랜치를 통한 CI/CD 동작 테스트 완료
- pr 리뷰 반영 추가

syjdjr committed:
Merge branch 'develop' into feat/#5-cicd
